source/usage-examples/command.txt

```diff
@@ -5,23 +5,33 @@ Run a Command
 .. default-domain:: mongodb
 
 You can run all raw database operations, not including
-:manual:`CRUD </crud/>` operations, using the :node-api:`command()
-<Admin.html#~command>` method. Call the ``command()`` method with
-your command object on an instance of ``Admin`` for diagnostic and
+:manual:`CRUD </crud/>` operations, using the :node-api:`db.command()
+<Db.html#~command>` method. Call the ``command()`` method with
+your command object on an instance of a database for diagnostic and
 administrative tasks such as fetching server stats or initializing a replica
 set.
 
 .. note::
     Use :manual:`collection methods </reference/method/js-collection>`
     instead of raw database commands whenever possible.
 
-Create an object to specify options. Set the ``maxTimeMS`` field of this
-object to state the number of milliseconds to wait before terminating the
-query.
+You can specify additional options in the ``options`` object passed in
```

Comment:
This section looks confusingly linked to me. It mentions the `options` one can pass into `deleteMany`, and then links to the `bb.command()` (which links to `Db#command`). Then it tells you to refer to the `Admin#~resultCallback`. Again, I think we're falling into territory that is better documented by API documentation, rather than duplicating that effort (incorrectly) here. 

I'm going to use the hashtag #dont-duplicate-api-documentation wherever I see this below.

Reply:
To address the comment about duplication of API documentation:
I agree there are some cases in which we can pare back some details, but we also want to ensure that we communicate what our team has identified as essential information without assuming the user reads and understands the content of all the links. 
In each of our Usage Examples, our team decided that essential information consisted of:
1) Method description and purpose
2) Required parameters and some options
3) Return value, if any
4) Commonly-encountered error cases and what may have caused them
I'll address each case and identify which ones I'd prefer to keep and a modified version that might address your concern.

Reply:
As mentioned in another comment, I'm eliminating #3 and providing a note at the top to refer to API documentation as suggested.